packages/polka/src/index.ts

```diff
@@ -81,7 +83,10 @@ export default class Polka implements IModule, IServerHandler {
     const polkaConfig = config?.get<PolkaConfig>(this.name);
 
     const app = polka({
-      onError: (err, _req, res) => {
+      onError: (err, req, res) => {
+        const traceId = req.traceId || uuidv4();
```

Comment:
We shouldn't be generating traceId here.

Reply:
Why not?